packages/cmf/src/cmfConnect.js

```diff
@@ -190,8 +190,31 @@ export default function cmfConnect({
 	mapStateToProps,
 	mapDispatchToProps,
 	mergeProps,
+	omitCMFProps = false, // will be removed and considered to true in 2.0.0
```

Comment:
Not sure if we should define this property.
Your wrote a codemod to adapt the code. Let's consider that we ask for no cmf props, and only use the detailed one. The codemod will ensure that the existing code works like today